meta-balena-common/recipes-bsp/grub/grub-efi_%.bbappend

```diff
@@ -0,0 +1,9 @@
+# We don't want grub modules in our sysroot, remove the entire prefix after do_deploy
```

Comment:
Don't we need to make it pull grub-conf also like we do for legacy grub?
`DEPENDS_append_class-target = " grub-conf"`

Reply:
It works currently without this, maybe because `grub-efi` RDEPENDS on `grub-common`? However, the original balena-intel `grub-efi` recipe does:

```
DEPENDS_append_class-target = " grub-conf"
RDEPENDS_${PN}_class-target_remove = "grub-common"
```

 It's unclear to me what the purpose of this is, @alexgg Could you weigh in on this? It looks like the original recipe removed `grub-common` from RDEPENDS and added `grub-conf`. Looking at poky's `grub_2.04.bb`, it looks like `grub-common` contains all the utilities and supporting files for grub, which we probably don't need.